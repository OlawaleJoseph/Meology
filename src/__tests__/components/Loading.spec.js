import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../../components/Loading';

describe('Loading component Snapshot', () => {
  it('builds loader correctly', () => {
    const tree = renderer.create(
      <Loader />,
    );
    expect(tree).toMatchSnapshot();
  });
});
