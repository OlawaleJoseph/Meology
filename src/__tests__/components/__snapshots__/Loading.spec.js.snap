// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component Snapshot builds loader correctly 1`] = `
<div
  className="loader"
/>
`;
